docs/sources/panels/panels-overview.md

```diff
@@ -12,9 +12,13 @@ draft = "true"
 
 The *panel* is the basic visualization building block in Grafana. Each panel has a query editor specific to the data source selected in the panel. The query editor allows you to extract the perfect visualization to display on the panel.
 
+With the exception of a few special use panels, a panel is a visual representation of one or more queries. The queries display data over time. This can range from temperature fluctuations to current server status to a list of logs or alerts.
```

Comment:
Is it always time series that is displayed? "The queries display data over time"

Reply:
Can be tabular data too but data over time is definitely the most common.